watertap/unit_models/translators/translator_asm2d_adm1.py

```diff
@@ -796,10 +880,10 @@ def SMg_AS6(blk, t):
             def SMg_output(blk, t):
                 return blk.properties_out[t].conc_mass_comp["S_Mg"] == blk.SMg_AS6[t]
 
-        # ---------------------------------------DecaySwitch.off-------------------------------------------------------#
+        # ---------------------------------------BioP.off-------------------------------------------------------#
 
         # -----------------------------------------Step 4--------------------------------------------------------------#
-        elif self.config.inlet_reaction_package.config.decay_switch == DecaySwitch.off:
+        elif self.config.bio_P == BioP.off:
```

Comment:
I couldn't see it with all the truncating that Github does, but it could be good to have an `else` here or following this. If the logic really only comes down to Bio_P being True or False, then this could just be `else`.

Reply:
I don't have an else - I'll add it here